sw/host/opentitanlib/src/util/vmem/mod.rs

```diff
@@ -80,8 +80,8 @@ mod test {
 
     #[test]
     fn vmem_data() {
-        let vmem = Vmem::from_str("@42 12 23 34 @84 @96 45").unwrap();
-        let expected = [(0x42, 0x12), (0x46, 0x23), (0x4a, 0x34), (0x96, 0x45)]
+        let vmem = Vmem::from_str("@10 12 23 34 @20 @26 45").unwrap();
+        let expected = [(0x40, 0x12), (0x44, 0x23), (0x80, 0x34), (0x98, 0x45)]
```

Comment:
Thanks for fixing this, I like the way you've done it! Byte addresses at the point of use make more sense.

I would expect this address to be `0x48` rather than `0x80` though. The `@20` address was meant to show that empty sections aren't visible by the time you get to the iterator.

```suggestion
        let vmem = Vmem::from_str("@10 12 23 34 @20 @26 45").unwrap();
        let expected = [(0x40, 0x12), (0x44, 0x23), (0x48, 0x34), (0x98, 0x45)]
```